draft-ietf-rats-eat.md

```diff
@@ -417,23 +417,26 @@ Endorsement:
 
 # Top-Level Token Definition
 
-An EAT is a "message", a "token", or such whose content is a Claims-Set about an entity or some number of entities.
-An EAT MUST always contains a Claims-Set.
+An EAT MUST contain a claims set about an entity packaged as a CWT {{RFC8392}} or JWT {{RFC7519}}. This specification extends {{RFC8392}} and {{RFC7519}} by defining additional claims for attestation. All definitions, requirements, creation and validation procedures, security considerations, IANA registrations, etc. from CWT and JWT apply to the EAT format.
 
-An EAT may be encoded in CBOR or JSON as defined here.
-While not encouraged, other documents may define EAT encoding in other formats.
+Authenticity and integrity protection MUST be provided for EATs. This document defines the use of CWT or JWT for this purpose.
+Extensions to this specification MAY use other methods of protection.
 
-EAT as defined here is always integrity and authenticity protected through use of CWT or JWT.
-Other token formats using other methods of protection may be defined outside this document.
+The identification of a protocol element as an EAT follows the general conventions used for CWTs and JWTs.
+Identification depends on the protocol carrying the EAT.
+In some cases it may be by media type (e.g., in a HTTP Content-Type field).
+In other cases it may be through use of CBOR tags.
+There is no fixed mechanism across all use cases.
 
-This document also defines the detatched EAT bundle ({{DEB}}), a bundle of some detached Claims-Sets and CWTs or JWTs that provide protection for the detached Claims-Set.
+This document also defines also defines a new top-level message, the detached EAT bundle (see {{DEB}}), which holds a collection of detached claims sets and an EAT that provides integrity and authenticity protection for them.
+Detached EAT bundles can be either CBOR or JSON encoded.
 
-The following CDDL defines the top-levels of an EAT token as a socket indicating future token formats may be defined.
-Any new format that plugs into this socket MUST be defined in a IETF standards track document.
-See {{CDDL_for_CWT}} for the CDDL definitions of a CWT and JWT.
+The following CDDL defines the top-level `$$EAT-CBOR-Tagged-Token`, `$$EAT-CBOR-Untagged-Token` and `$$EAT-JSON-Token-Formats` sockets, enabling future token formats to be defined.
+Any new format that plugs into one or more of these sockets MUST be defined in a IETF standards track document.
+Of particular use may be a token type that provides no direct authenticity or integrity protection for use with transports mechanisms that do provide the necessary security services {{UCCS}}.
```

Comment:
I'm fine with the reference as long as it doesn't slow down EAT.

We had this big discussion in Vienna that I thought concluded that the reference had to go because I didn't think you could refer to a draft document in a published RFC, normative or non-normative. If that's true then with this reference EAT would have to wait for UCCS to become an RFC. 

(And if we can reference UCCS, then maybe we should put back all the CDDL and other stuff for UCCS. There was a fair bit of it. The PR to remove the UCCS reference was large and complicated).

So first question to answer is whether a publish RFC can references an I-D.

Reply:
removed as was intent, I must have missed a save

Reply:
Actually, I'd failed to save a major change to my comment above that said it was OK to leave it in. ooops.

I found that you can have a non-normative reference to a I-D. The method for doing it is here: https://www.rfc-editor.org/rfc/rfc7322#section-4.8.6.4

I am OK either way. It might be kind of good to make the reference so the reader knows about it rather than hoping they'll come across it.

Reply:
Will put it back.